docs/advanced.md

```diff
@@ -41,6 +41,31 @@ with LoggingContext(user=request.user.id, _prefix="demo"):
 In the above message, the `user` contextual key would be outputted as `demo.user`, regardless of the value of the
 `WOODCHIPPER_KEY_PREFIX` environment variable.
 
+## Changing context logging levels
+
+Woodchipper supports changing the log level of entrance and exit logging for contexts, with the default log level of `"INFO"`.
+
+You can set the default log level for all context logs by setting the environment variable `WOODCHIPPER_CONTEXT_LOG_LEVEL` to
+the desired log level name. For example, if you set `WOODCHIPPER_CONTEXT_LOG_LEVEL` to `"DEBUG"` all logs created when entering or
+exiting a LoggingContext would be debug by default.
+
+You can also set the log level on a `LoggingContext`, overriding the default log level, with the `_log_level` kwarg. For
+example:
+
+```python
+with LoggingContext(user=request.user.id, _log_level="ERROR"):
```

Comment:
Maybe this is a python convention I'm not familiar with, but why use an underscore in front of the _log_level parameter? It's not a private attribute of a class, right?

Reply:
To indicate that it's not a context variable that should be added to the logging context but instead a configuration directive for `LoggingContext`.

Reply:
Ahh got it, thank you!